src/app/components/ScrollablePromo/Promo/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, string, oneOfType, object } from 'prop-types';
+import styled from '@emotion/styled';
+import pathOr from 'ramda/src/pathOr';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { getSerifRegular } from '@bbc/psammead-styles/font-styles';
+import { C_GREY_6, C_WHITE } from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_0_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { ServiceContext } from '#contexts/ServiceContext';
+import filterForBlockType from '#lib/utilities/blockHandlers';
+
+const C_GREY_8 = '#202224';
+
+const Link = styled.a`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => service && getSerifRegular(service)}
+  width: 100%;
+
+  text-overflow: ellipsis;
```

Comment:
Has this been asked for specifically?

Reply:
If so have we tested this with a screen reader?

Reply:
yes, the specications says that instead of overflowing the text should be truncated to 3 lines.

Reply:
Thanks for testing with our screen readers, and confirming this reads out all the text  when an ellipsis is displayed